scripts/dev/devimint-env.sh

```diff
@@ -20,6 +20,8 @@ function devimint_env {
   # For starship users, we can actually make the prompt distinct so there's
   # no confusion.
   export STARSHIP_CONFIG="${REPO_ROOT}/scripts/dev/devimint-env/starship.toml"
+  source "${REPO_ROOT}/scripts/dev/aliases.sh"
```

Comment:
for fish we can do `fish --init-command 'source aliases.sh`  but bash can't do much.

Reply:
PATH works for all shells, current and future ones.